src/app/body/create-new-organization/create-new-organization.component.ts

```diff
@@ -108,4 +109,17 @@ export class CreateNewOrganizationComponent implements OnInit {
   close() {
     this.location.back();
   }
+
+  checkOrgDomainAvailabilityLive() {
+    for (let i = 0; i < this.authService.allOrgDomains.length; i++) {
+      if (this.authService.allOrgDomains[i]==this.orgDomain.value) {
+        this.orgDomainsAvailable = false;
+        break;
+      }else {
+        this.orgDomainsAvailable = true;
+          this.enableLoader = false;
+      }
+  }
+  
+  }
```

Comment:
just use includes function instead of this for loop

Reply:
resolved